src/components/Tooltip/Tooltip.js

```diff
@@ -58,6 +63,7 @@ export default class Tooltip extends Component {
       showIcon,
       iconName,
       iconDescription,
+      menuOffset,
```

Comment:
When a user enters a value of `menuOffset = {left: 10}` that overwrites the object entirely, giving `menuOffset.top` a value of `undefined`. This breaks the view.